content/v2/tlds.markdown

```diff
@@ -47,6 +47,16 @@ List all TLDs.
 <%= pretty_print_fixture("/listTlds/success.http") %>
 ~~~
 
+### Sorting
+
+For general information about sorting, please refer to the [main guide](/v2/#sorting).
+
+Name | Description
+-----|------------
+`tld` | Sort TLDs by tld
```

Comment:
I think it's valuable as a way of knowing what's the ordering criteria to be expected from the endpoint. I'm 👍 on keeping it.